src/main/java/step3/domain/LottoManager.java

```diff
@@ -0,0 +1,84 @@
+package step3.domain;
+
+import static java.util.Collections.shuffle;
+import static step3.constant.LottoInfoConstant.LOTTO_ELEMENTS_SIZE;
+import static step3.constant.LottoInfoConstant.LOTTO_PRICE;
+import static step3.constant.LottoInfoConstant.LOTTO_VALID_ELEMENTS;
+import static step3.constant.enums.LottoReward.numberToLottoReward;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import step3.constant.enums.LottoReward;
+
+public class LottoManager {
+
+    private List<LottoTicket> lottoTickets = new ArrayList<>();
+
+
+    public LottoManager() {
+    }
+
+    public LottoManager(List<LottoTicket> lottoTickets) {
+        this.lottoTickets = lottoTickets;
+    }
+
+
+    public int buyRandomTicket(String money) {
+        if (!validMoney(money)) {
+            throw new IllegalArgumentException("돈은 " + LOTTO_PRICE + "이상 입력하셔야합니다");
+        }
+        for (int i = 0; i < Integer.parseInt(money) / LOTTO_PRICE; i++) {
+            lottoTickets.add(makeRandomLottoTicket());
+        }
+        return lottoTickets.size();
+    }
+
+
+    private boolean validMoney(String money) {
+        boolean validResult = true;
+        try {
+            validResult = Integer.parseInt(money) >= LOTTO_PRICE;
+        } catch (NumberFormatException e) {
+            validResult = false;
+        } finally {
+            return validResult;
+        }
```

Comment:
`validResult`를 지역 변수로 선언해서 값을 변경 후 `finally` 구문에서 반환하고 있는데요.
다음과 같이 수정하면 지역 변수를 사용하지 않고 값을 반환할 수 있습니다.
```suggestion
        try {
            return Integer.parseInt(money) >= LOTTO_PRICE;
        } catch (NumberFormatException e) {
            return false;
        }
```